rosdep/python.yaml

```diff
@@ -7650,6 +7650,15 @@ python3-pyside2.qtopengl:
     '*': [python3-pyside2.qtopengl]
     bionic: null
     xenial: null
+python3-pyside2.qtquick:
+  arch: [pyside2]
+  debian: [python3-pyside2.qtquick]
+  fedora: [python3-pyside2]
+  gentoo: [dev-python/pyside2
```

Comment:
```suggestion
  gentoo: [dev-python/pyside2]
```

Reply:
I think CI is failing because of this, LGTM with it corrected.

Reply:
Added missing bracket